runway/module/serverless.py

```diff
@@ -16,33 +15,6 @@
 LOGGER = logging.getLogger('runway')
 
 
-def gen_sls_config_files(stage, region):
```

Comment:
Moved these inside the class to take advantage of `folder` and to remove some params.